.circleci/config.yml

```diff
@@ -6,37 +6,140 @@ anchor_docker: &anchor_docker
       environment:
         TERM: dumb
 
+anchor_auth_vars: &anchor_auth_vars
+  environment:
+    GCLOUD_PROJECT: gcloud-devel
+    GOOGLE_APPLICATION_CREDENTIALS: .circleci/it-service-account.json
```

Comment:
Even though this is encrypted, is it ok to host in a public repo?

Reply:
I think so. Other languages are already doing it. (I know, great excuse, isn't it?)